web/src/js/ducks/modes/local.ts

```diff
@@ -0,0 +1,103 @@
+import { ModeState, updateMode } from "../modes";
+import {
+    RECEIVE as RECEIVE_OPTIONS,
+    UPDATE as UPDATE_OPTIONS,
+} from "../options";
+import { getModesOfType } from "./utils";
+
+export const TOGGLE_LOCAL = "TOGGLE_LOCAL";
+export const SET_APPLICATIONS = "SET_APPLICATIONS";
```

Comment:
We have two options here: 

1. Prefix events with something like `MODE_LOCAL_` so that we don't get name collisions.
2. Use Redux's createSlice API that avoids this footgun. 😛